### PR TITLE
feat: upgrade MongoDB library to support MongoDB 5.1+

### PR DESCRIPTION
**Pull request by kjellwinblad**
Question: Since the enterprise branch has other apps that depends on MongoDB, I guess the enterprise branch has to be updated as well, right?

### CODE REVIEW

File: changes/v5.0.14/feat-8329.en.md [hunk 1]
```diff
@@ -0,0 +1 @@
+The MongoDB library has been upgraded to support MongoDB 5.1+
```
Comment (thalesmg) [inline changes/v5.0.14/feat-8329.en.md:1]: Reminder: needs a ZH translation as well.

**Review by thalesmg — APPROVED** [review]

File: changes/v5.0.14/feat-8329.en.md (hunk 1 above)
Comment (lafirest) [reply changes/v5.0.14/feat-8329.en.md:1]: MongoDB 驱动现在已经升级到 MongoDB 5.1+ 了。

Comment (kjellwinblad) [reply changes/v5.0.14/feat-8329.en.md:1]: Added a new commit with your Chinese translation

**Comment by kjellwinblad** [comment]
@thalesmg need a new approval after adding Chinese translation. Thanks.

**Review by thalesmg — APPROVED** [review]

**Pull request merged**